index/models.py

```diff
@@ -122,15 +122,14 @@ def add_info_other(self, info):
 class Petition(models.Model):
     title = models.CharField(max_length=200)  # Title of petition
     description = models.CharField(max_length=4000)  # The paragraph description
-    ID = models.IntegerField(primary_key=True)
     archived = models.BooleanField(default=False)
     hidden = models.BooleanField(default=False)
 
     created_date = models.DateTimeField(
         db_index=True, default=timezone.now
     )  # Files the date created
     expected_sig = models.IntegerField(
```

Comment:
Instead of having `expected_sig` as a field in the `Petition` model, we could read it from an environment variable or make it a global variable. That way, as you pointed out, if the number changes, all of the existing petitions won't break. Additionally, I think storing this as a field with every petition object is unnecessary.

Reply:
I think the signature threshold should be in the model. This could be useful if the required number of signatures for recognition gets raised or lowered in the future, since a new threshold should only apply to new petitions. Then the already-existing petitions would still be operating with the old threshold and any UI elements could use the old threshold to show e.g. progress bars toward recognition.

Reply:
True, that's a good point.